src/Reducers/Settings.js

```diff
@@ -3,11 +3,30 @@ import * as Settings from '../Actions/Settings'
 const initialState = {}
 
 export default function (state = initialState, action) {
+  let nextState = JSON.parse(JSON.stringify(state))
```

Comment:
`JSON.parse(JSON.stringify(...))` is pretty inefficient. We typically use `Object.assign({}, state, updatedState)` but `_.merge({}, state, updatedState)` is even better.

Reply:
Also it's slightly more idiomatic if you do this inline w/ your case statements rather than create an object and then modify it in your cases. Up to you as to whether you want to change this, though.